### PR TITLE
Fixes format template modifiers

### LINKED ISSUE #1781: Backend formatdate and formatdatetime throwing error
## Type
- Non critical bug
## Problem description

> Backend formatdate and formatdatetime throwing error
## Steps to reproduce

> When using `{{ item.createdOn|formatdate }}` or `{{ item.createdOn|formatdatetime }}` an error is thrown.

```
Attempted to call an undefined method named "formatDate" of class "Backend\Core\Engine\TemplateModifiers".
Did you mean to call "formatTime"?
500 Internal Server Error - UndefinedMethodException
```

`{{ item.createdOn|formattime }}` is working!
## Expected behavior / Proposed solutions

> That it works

### PR DESCRIPTION
**Pull request by jeroendesloovere**
Fixes #1781

### CODE REVIEW

File: src/Backend/Cache/Analytics/Index_1426692958178.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+done
```
Comment (carakas) [inline src/Backend/Cache/Analytics/Index_1426692958178.txt]: This shouldn't be here

File: src/Backend/Cache/Analytics/1425942000_1426633200.xml [hunk 2]
```diff
@@ -0,0 +1,238 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (carakas) [inline src/Backend/Cache/Analytics/1425942000_1426633200.xml]: this shouldn't be here

File: composer.lock [hunk 3]
```diff
@@ -1170,6 +1170,90 @@
             "time": "2015-12-19 20:16:43"
         },
         {
+            "name": "league/flysystem",
```
Comment (carakas) [inline composer.lock]: wasn't this already added?

File: src/Backend/Core/Engine/TemplateModifiers.php [hunk 4]
```diff
@@ -92,6 +92,40 @@ public static function formatNumber($number, $decimals = null)
...
+     *
+     * @return string
+     */
+    public static function formatDateTime($var)
+    {
+        // get setting
+        $format = Authentication::getUser()->getSetting('datetime_format');
+
+        // format the date
+        return \SpoonDate::getDate($format, (int) $var, BackendLanguage::getInterfaceLanguage());
```
Comment (carakas) [inline src/Backend/Core/Engine/TemplateModifiers.php]: maybe add a use statement for this

File: composer.lock (hunk 3 above)
Comment (carakas) [inline composer.lock]: you actually messed up composer install
Your requirements could not be resolved to an installable set of packages.

**Comment by carakas** [comment]
Are they also available in the frontend? because they are also loaded there

File: src/Frontend/Core/Engine/TemplateModifiers.php [hunk 5]
```diff
@@ -11,7 +11,6 @@
 
 use Frontend\Core\Engine\Model as FrontendModel;
 use Frontend\Core\Engine\Block\Widget as FrontendBlockWidget;
-use Frontend\Core\Language\Language;
```
Comment (carakas) [inline src/Frontend/Core/Engine/TemplateModifiers.php]: Wrong implementation, the other one will throw deprecation warnings so you should create an alias for it and use that alias

**Comment by carakas** [comment]
@jeroendesloovere can you look at the feedback and the failed ci build

File: src/Frontend/Core/Engine/TemplateModifiers.php (hunk 5 above)
Comment (jeroendesloovere) [inline src/Frontend/Core/Engine/TemplateModifiers.php]: There is no Language left in this class, so I removed this line...

Comment (jeroendesloovere) [inline src/Frontend/Core/Engine/TemplateModifiers.php]: I'm using `LANGUAGE`

Comment (carakas) [inline src/Frontend/Core/Engine/TemplateModifiers.php]: you should use `Frontend\Core\Language\Locale`

Comment (jeroendesloovere) [inline src/Frontend/Core/Engine/TemplateModifiers.php]: Done.

**Pull request merged**